src/User32.Desktop/User32+DLGITEMTEMPLATE.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.

namespace PInvoke
{
    using System.Runtime.InteropServices;

    /// <content>
    /// Contains the <see cref="DLGITEMTEMPLATE"/> nested type.
    /// </content>
    public partial class User32
    {
        /// <summary>
+        ///     Defines the dimensions and style of a control in a dialog box. One or more of these structures are combined with a
+        ///     <see cref="DLGTEMPLATE" /> structure to form a standard template for a dialog box.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential, Pack = 2)]
+        public struct DLGITEMTEMPLATE
+        {
+            /// <summary>
+            ///     The style of the control. This member can be a combination of window style values (such as WS_BORDER) and one or
+            ///     more of the control style values (such as BS_PUSHBUTTON and ES_LEFT).
+            /// </summary>
+            public int style;
+
+            /// <summary>
+            ///     The extended styles for a window. This member is not used to create controls in dialog boxes, but applications that
+            ///     use dialog box templates can use it to create other types of windows.
+            /// </summary>

```

Comment:
In this specific case there is a comment before the declaration in the original header file (`WinUser.h`) that specify this :

```c
/*
 * WARNING:
 * The following structures must NOT be DWORD padded because they are
 * followed by strings, etc that do not have to be DWORD aligned.
 */
#include <pshpack2.h>

/* ... */

#include <poppack.h> /* Resume normal packing */
```

The `pshpack2.h` file seem to be used in a dozen of places in the whole SDK so it's not a very frequent occurrence.